test/test_cellstotex.py

```diff
@@ -0,0 +1,107 @@
+import os
+from mathics.core.parser import parse, MathicsSingleLineFeeder
+from mathics.core.definitions import Definitions
+from mathics.core.evaluation import Evaluation
+from mathics.core.expression import Symbol
+import pytest
+import urllib.request
+import unittest
+
+external_url = (
+    "https://raw.githubusercontent.com/jkuczm/MathematicaCellsToTeX/master/NoInstall.m"
+)
+
+pytestmark = pytest.mark.skipif(os.getenv("SKIP_CELLSTOTEX", None) is not None,
+                                reason="SKIP_CELLSTOTEX environment variable set")
+
+try:
+    http_code = urllib.request.urlopen(external_url).getcode()
+except:
+    url_reachable = False
+else:
+    url_reachable = http_code in (200,) # add other 2xx or 3xx's?
+
+definitions = Definitions(add_builtin=True)
+evaluation = Evaluation(definitions=definitions, catch_interrupt=False)
+set_versionnumber = 'Unprotect[$VersionNumber];$VersionNumber=11;Protect[$VersionNumber];'
+import_url = 'Import@"%s"' % external_url
+
+
+def _evaluate(str_expression):
+    expr = parse(definitions, MathicsSingleLineFeeder(str_expression))
+    return expr.evaluate(evaluation)
+
+_evaluate('LoadModule["pymathics.asy"]')
+
+def test_load():
+    str_expected1 = "{}"
+    message1 = ""
+    _evaluate(set_versionnumber)
+    result1 = _evaluate(import_url)
+    expected1 = _evaluate(str_expected1)
+
+
+    if message1:
+        assert result1 == expected1, message1
+    else:
+        assert result1 == expected1
+
+    result2 = _evaluate('Names["CellsToTeX`*"]')
+    expected2 = _evaluate('{"CellToTeX", "CellsToTeXException", "CellsToTeXPreamble"}')
+    print(result2)
+    assert result2 == expected2
+
+
+@pytest.mark.skipif(not url_reachable, reason="skipping since we can't reach %s" % external_url)
+@pytest.mark.xfail
+def test_load_and_run():
+    print("load and run")
+    str_expected0 = "None"
```

Comment:
All of this should be redone using the helper. 

Also in the future tests like this should be moved to mathics_omnibus.